searchlib/src/vespa/searchlib/docstore/logdatastore.cpp

```diff
@@ -422,10 +382,24 @@ LogDataStore::findNextToCompact(double bloatLimit, double spreadLimit, bool prio
 }
 
 void
-LogDataStore::compactWorst(double bloatLimit, double spreadLimit, bool prioritizeDiskBloat) {
-    auto worst = findNextToCompact(bloatLimit, spreadLimit, prioritizeDiskBloat);
-    if (worst.first) {
-        compactFile(worst.second);
+LogDataStore::compactWorst(uint64_t syncToken, bool compactDiskBloat) {
+    uint64_t usage = getDiskFootprint();
+    uint64_t bloat = getDiskBloat();
+    const char * reason = compactDiskBloat ? "bloat" : "spread";
+    LOG(debug, "%s", bloatMsg(bloat, usage).c_str());
+    const bool doCompact = (_fileChunks.size() > 1);
+    if (doCompact) {
+        LOG(info, "%s. Will compact due to %s", reason, bloatMsg(bloat, usage).c_str());
```

Comment:
Consider "Will compact due to %s: %s" instead. Then it matches better with L400.

Reply:
Fixed